### PR TITLE
Add texture plugin to support ktx loader 

### PR DESCRIPTION
**Pull request by zhuxudong**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/galacean/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/loader/src/gltf/parser/GLTFTextureParser.ts [hunk 1]
```diff
@@ -18,47 +26,62 @@ export class GLTFTextureParser extends GLTFParser {
...
-              }
-              const bufferTextureRestoreInfo = new BufferTextureRestoreInfo(texture, bufferView, mimeType);
-              context.contentRestorer.bufferTextures.push(bufferTextureRestoreInfo);
-
-              return texture;
-            });
+            }
           }
+
+          return Promise.all([texture]).then((textures) => {
```
Comment (gz65555) [inline packages/loader/src/gltf/parser/GLTFTextureParser.ts:80]: Why use `Promise.all`？

Comment (zhuxudong) [reply packages/loader/src/gltf/parser/GLTFTextureParser.ts:80]: done

**Review by gz65555 — APPROVED** [review]

File: packages/loader/src/gltf/parser/GLTFTextureParser.ts [hunk 2]
```diff
@@ -18,47 +26,61 @@ export class GLTFTextureParser extends GLTFParser {
...
-              .load<Texture2D>({
-                url: Utils.resolveAbsoluteUrl(url, uri),
-                type: type
-              })
-              .then((texture) => {
-                if (!texture.name) {
-                  texture.name = textureName || imageName || `texture_${index}`;
-                }
+
+          let texture = <Texture | Promise<Texture> | AssetPromise<Texture>>(
```
Comment (gz65555) [inline packages/loader/src/gltf/parser/GLTFTextureParser.ts:35]: 用 `<Texture | Promise<Texture>` 即可

Comment (gz65555) [reply packages/loader/src/gltf/parser/GLTFTextureParser.ts:35]: `AssetPromise extends Promise`

**Pull request closed without merging**